docs/pages/patterns/comments.vue

```diff
@@ -95,13 +95,69 @@
 
 // Delete Payload
 {
-	id: Number
-	message: String
-	parentId: Number
 }
 
 ```
 
+## Parsing HTML 
+
+KtComment will escape all tags by default but you can opt out and pass your own parser by using the parser prop 
+
+> Remember to still escape tags to prevent XHR attacks,
```

Comment:
```suggestion
> Remember to **escape malicious tags** to prevent [Cross-site-scripting](https://en.wikipedia.org/wiki/Cross-site_scripting) attacks,
```